pr_agent/algo/pr_processing.py

```diff
@@ -333,7 +333,7 @@ def generate_full_patch(convert_hunks_to_line_numbers, file_dict, max_tokens_mod
     return total_tokens, patches, remaining_files_list_new, files_in_patch_list
 
 
-async def retry_with_fallback_models(f: Callable, model_type: ModelType = ModelType.WEAK):
+async def retry_with_fallback_models(f: Callable, model_type: ModelType = ModelType.REGULAR):
```

Comment:
some additional adjustments 3:

This is a fix. The default should be REGULAR

Reply:
I'm wondering because my commit already set the default model to `REGULAR` https://github.com/KennyDizi/pr-agent/commit/6352e6e3bfcccf5a4e75f68a7f9e1714eb56bedd

Reply:
I'm wondering because my commit already set the default model to `REGULAR` as your suggestion https://github.com/KennyDizi/pr-agent/commit/6352e6e3bfcccf5a4e75f68a7f9e1714eb56bedd